code/game/objects/items/trader.dm

```diff
@@ -752,3 +752,37 @@
 		name = "[t]'s ID card ([assignment])"
 	else
 		return
+
+/obj/item/weapon/mech_expansion_kit
+	name = "exosuit expansion kit"
+	desc = "All the equipment you need to replace that useless legroom with a useful bonus equipment slot on your mech."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "modkit"
+	flags = FPRINT
+	siemens_coefficient = 0
+	w_class = W_CLASS_SMALL
+	var/working = FALSE
```

Comment:
Is this really necessary? The do_after should fail if the item in question disappears so you really can't upgrade the mecha several times with the same item

Reply:
it isn't necessary but it allows me to have a special fail message